sci-electronics/kicad/kicad-4.0.6.ebuild

```diff
@@ -0,0 +1,198 @@
+src_prepare() {
+	xdg_src_prepare
+
+	# Patch to work with >=boost 1.61
+	eapply "${FILESDIR}/${PN}-boost-1.61.patch"
+
+	# remove all the non unix file endings
+	while IFS="" read -d $'\0' -r f; do
+		edos2unix "${f}"
+	done < <(find "${S}" -type f -name "*.desktop" -print0)
+
+	# Remove cvpcb desktop file while it does nothing
+	rm "${WORKDIR}/${P}/resources/linux/mime/applications/cvpcb.desktop" || die
+
+	# Handle optional minimal install.
+	if use minimal; then
+		# remove templates as they are not needed to run binaries
+		sed -e '/add_subdirectory( template )/d' -i CMakeLists.txt || die
+	else
+		# create a link to the parts library in the main project folder
+		ln -s "${WORKDIR}/kicad-library-${PV}" "${S}/${PN}-library" || die
+		# create a link to the footprints library and add cmake build rule for it
+		ln -s "${WORKDIR}/${PN}-footprints-${PV}" "${S}/${PN}-footprints" || die
+		cp "${FILESDIR}/${PN}-footprints-cmakelists.txt" "${WORKDIR}/${PN}-footprints-${PV}/CMakeLists.txt" || die
+		# add the libraries directory to cmake as a subproject to build
+		sed "/add_subdirectory( bitmaps_png )/a add_subdirectory( ${PN}-library )" -i CMakeLists.txt || die
+		# add the footprints directory to cmake as a subproject to build
+		sed "/add_subdirectory( ${PN}-library )/a add_subdirectory( ${PN}-footprints )" -i CMakeLists.txt || die
+		# remove duplicate uninstall directions for the library module
+		sed '/make uninstall/,/# /d' -i ${PN}-library/CMakeLists.txt || die
+	fi
+
+	# Add internationalization for the GUI
+	if use i18n; then
+		# create a link to the translations library in the main project folder
+		ln -s "${WORKDIR}/${P}-i18n" "${S}/${PN}-i18n" || die
+		# Remove unused languages. Project generates only languages specified in the
+		# file in LINGUAS in the subproject folder. By default all languages are added
+		# so we sed out the unused ones based on the user linguas_* settings.
+		local lang
+		for lang in ${LANGS}; do
+			if ! use linguas_${lang}; then
+				sed "/${lang}/d" -i ${PN}-i18n/LINGUAS || die
+			fi
+		done
+		# cmakelists does not respect our build dir variables, so make it point to the right location
+		sed "s|\${CMAKE_BINARY_DIR}|${WORKDIR}/${P}_build|g" -i ${PN}-i18n/CMakeLists.txt || die
+		# we also make from the master project so the source dir is understood incorretly, replace that too
+		sed "s|\${CMAKE_SOURCE_DIR}/\${LANG}|\${CMAKE_SOURCE_DIR}/${PN}-i18n/\${LANG}|g" -i ${PN}-i18n/CMakeLists.txt || die
+		# add the translations directory to cmake as a subproject to build
+		sed "/add_subdirectory( bitmaps_png )/a add_subdirectory( ${PN}-i18n )" -i CMakeLists.txt || die
+		# remove duplicate uninstall directions for the translation module
+		sed '/make uninstall/,$d' -i ${PN}-i18n/CMakeLists.txt || die
+	fi
+
+	# Install examples in the right place if requested
+	if use examples; then
+		# install demos into the examples folder too
+		sed -e 's:${KICAD_DATA}/demos:${KICAD_DOCS}/examples:' -i CMakeLists.txt || die
```

Comment:
I will double check, but I think that's where the main application expects the files to be.